sdk-utils/src/main/kotlin/com/rakuten/tech/mobile/sdkutils/StaticInfoUtil.kt

```diff
@@ -0,0 +1,111 @@
+package com.rakuten.tech.mobile.sdkutils
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import org.json.JSONObject
+
+/**
+ * This utility class provides application information.
+ */
+object StaticInfoUtil {
+    /**
+     * Get the application information.
+     *
+     * @return return the app info map data based on AppInfo and BuildInfo keys
+     */
+    fun getAppInfo(): String {
+        val infoMap = HashMap<String, Any?>()
+
+        // Populate the System properties
+        infoMap[AppInfoKey.PROPERTIES] = getSystemProperties()
+
+        // Populate all packages
+        infoMap[AppInfoKey.PACKAGES] = getPackages()
+
+        // Populate build info
+        infoMap[BuildInfoKey.BUILD] = getBuildMap()
+
+        return JSONObject(infoMap).toString()
+    }
+
+    /**
+     * Get the system properties.
+     *
+     * @return the system properties map
+     */
+    fun getSystemProperties(): HashMap<String, String> {
+        val propertiesHashMap = HashMap<String, String>()
+        val properties = System.getProperties()
+        for (name in properties.stringPropertyNames()) {
```

Comment:
are the app package name, app version name, app version code included in this? if not, better if we add them.

Reply:
We already have a **AppInfo.kt** which gives the app name, version name. Should we add this here as well

Reply:
aah got it.
thanks